doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -1,6 +1,6 @@
 .. _thecoqcommands:
 
-Coq commands
+The Rocq Prover commands
 ====================
```

Comment:
Apparently the issue is here
```suggestion
The Rocq Prover commands
========================
```

Reply:
I fixed this one and many other similar cases. Sorry I didn't run the build locally the first time I pushed.

Reply:
No worries, that's why we have CI